node_scenarios/node_scenarios_example.yml

```diff
@@ -0,0 +1,16 @@
+node_scenarios:
+  - actions: 
+    - node_stop_scenario
+    - node_start_scenario
+    node_name:
+    label_selector: node-role.kubernetes.io/worker
+    instance_kill_count: 1
+    timeout: 120
+    cloud_type: aws
+  - actions:
+    - node_reboot_scenario
+    node_name:
+    label_selector: node-role.kubernetes.io/infra
+    instance_kill_count: 1
```

Comment:
Maybe rename instance_kill_count to instance_count since the action can also be a reboot/stop?